pombola/core/templates/core/object_base.html

```diff
@@ -11,6 +11,22 @@
 
         {% block object_title %}
           <h1>{{ object.title }} {{ object.name }}</h1>
+          {% for hist in object.org_history_old.all %}
+              <span class="committee-history">
+                  {{ hist.date_changed|date:"d M Y" }} | Became:
+                 <a href="{{ hist.new_organisation.get_absolute_url }}">
+                    {{ hist.new_organisation }}
```

Comment:
```suggestion
                    {{ hist.new_organisation.name }}
```

or whatever - we shouldn't rely on the org.__str__ being the correct thing for presentation here